indra/llcharacter/CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ if (LL_TESTS)
   include(LLAddBuildTest)
   # UNIT TESTS
   SET(llcharacter_TEST_SOURCE_FILES
-    #llik.cpp
+    llik.cpp
```

Comment:
I'm re-enabling the IK unit test projbect although most of the unit tests are temporarily disabled.